Integrations/AttackIQFireDrill/AttackIQFireDrill.py

```diff
@@ -525,6 +529,137 @@ def run_all_tests_in_assessment_command():
+            'User': asset.get('user', ''),
+            'Status': asset.get('status', '')
+        }
+        groups = []
+        for group in asset.get('asset_groups', []):
+            temp_group = {
+                "ID": group.get('id'),
+                "Name": group.get('name')
+            }
+            groups.append(temp_group)
+        asset_dict['Groups'] = groups
+        assets.append(asset_dict)
+    ec = {
+        "AttackIQ.Asset(val.ID && val.ID === obj.ID)": assets
+    }
+    hr = tableToMarkdown("Assets:", assets, ['ID', 'Hostname', 'IPv4', 'MacAddress', 'Domain',
+                                             'Description', 'User', 'Status'])
+
+    return_outputs(hr, ec, res)
+
+
+@logger
+def create_assessment_command():
+    """
+    name - The name of the assesment to create.
+
+    Returns:
+
+    """
+    body = {
+        "project_name": demisto.args().get('name'),
+        "template": demisto.args().get('template_id')
+    }
+
+    try:
+        res = http_request('POST', '/v1/assessments/project_from_template', data=json.dumps(body))
+    except Exception as e:
+        raise ValueError(f"Could not create an assessment. Specifically: {str(e)}")
+
+    assessment_id = res.get('project_id')
+    try:
+        raw_assessments = get_assessments(assessment_id=assessment_id)
+        assessments_res = build_transformed_dict(raw_assessments, ASSESSMENTS_TRANS)
+        hr = tableToMarkdown(f'Created Assessment: {assessment_id} successfully.', assessments_res,
+                             headers=['Id', 'Name', 'Description', 'User', 'Created', 'Modified'])
+        return_outputs(hr, {'AttackIQ.Assessment(val.Id === obj.Id)': assessments_res}, raw_assessments)
+    except HTTPError as e:
+        return_error(create_invalid_id_err_msg(str(e), ['403']))
+
+
+@logger
+def add_assets_to_assessment():
+    assessment_id = demisto.args().get('assessment_id')
+    assets = demisto.args().get('assets')
+    asset_groups = demisto.args().get('asset_groups')
+
+    data = {}
+    if assets:
+        data['assets'] = assets
+    if asset_groups:
+        data['asset_groups'] = asset_groups
+
+    if data == {}:
+        raise ValueError(f"No asset or asset groups were specified.")
+    try:
+        res = http_request('POST', f'/v1/assessments/{assessment_id}/update_defaults', data=json.dumps(data))
+        demisto.results(res.get('message', ''))
+    except Exception:
+        raise ValueError("Could not find either the assessment or one of the assets/asset groups.")
+
+
+@logger
+def delete_assessment_command():
+    assessment_id = demisto.args().get('assessment_id')
+    try:
+        http_request('DELETE', f'/v1/assessments/{assessment_id}')
+        demisto.results(f"Deleted assessment {assessment_id} successfully.")
+    except Exception as e:
+        if '403' in str(e):
+            raise ValueError(f"Could not find the assessment {assessment_id}")
+        else:
+            raise e
```

Comment:
```suggestion
            raise
```

Reply:
"raise" raises the exception? (e is redundant here, not forgotten?)